src/main/resources/application.yml

```diff
@@ -279,7 +279,7 @@ issuebot:
           - jzheaux
           - rwinch
           - sjohnr
-      - organization: spring-projects-experimental
```

Comment:
Should this have been a change to line 274 instead?

Reply:
Oops. Thanks for catching that! I'll update it.